src/expandableCalendar/week.js

```diff
@@ -3,15 +3,13 @@ import XDate from 'xdate';
 import React, {Component} from 'react';
 import {View} from 'react-native';
 import dateutils from '../dateutils';
-import {xdateToData, parseDate} from '../interface';
-import {SELECT_DATE_SLOT} from '../testIDs';
+import {parseDate} from '../interface';
+import {extractComponentProps} from '../component-updater';
 import styleConstructor from './style';
 import Calendar from '../calendar';
-import Day from '../calendar/day/basic';
-import PeriodDay from '../calendar/day/period';
-import MultiDotDay from '../calendar/day/multi-dot';
-import MultiPeriodDay from '../calendar/day/multi-period';
-import CustomDay from '../calendar/day/custom';
+import Day from '../calendar/day/index';
+// import BasicDay from '../calendar/day/basic';
```

Comment:
Comment

Reply:
I left it to when I'll add the week numbers support.